riotfile.py

```diff
@@ -131,6 +131,21 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "mypy": "<0.900",
             },
         ),
+        Venv(
+            pys=["3"],
+            pkgs={"codespell": "==2.1.0"},
+            venvs=[
+                Venv(
+                    name="codespell",
+                    command="codespell --ignore-words-list='dne,fo,medias,ment,nin,ot,setttings,statics' "
+                    "--skip='ddtrace/vendor' ddtrace/ tests/",
+                ),
+                Venv(
+                    name="hook-codespell",
+                    command="codespell {cmdargs}",
```

Comment:
we still need the `ignore-words-list` and `skip` here?

Reply:
looks like we can setup a config file as well for the ignore words, skips, etc

https://github.com/codespell-project/codespell/#using-a-config-file